tavla/app/(admin)/utils/index.ts

```diff
@@ -72,9 +72,15 @@ export function getFormFeedbackForError(
         case 'auth/weak-password':
             return {
                 form_type: 'password',
-                feedback: 'Passordet er for svakt.',
+                feedback: 'Passordet er for kort – det må være minst 6 tegn.,
```

Comment:
upsi manglet et tegn
```suggestion
                feedback: 'Passordet er for kort – det må være minst 6 tegn.',
```